src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -97,14 +99,35 @@ export default class AircraftInstanceModel {
         this.approachDistance = 0;      // Distance longitudinally from the threshold
         this.radial       = 0;          // Angle from airport center to aircraft
         this.distance     = 0;          //
-        this.destination  = null;       // Destination they're flying to
+
+        /**
+         * The origin ariport for an aircraft
+         *
+         * This will only be populated for dpearture aircraft
+         *
+         * @property origin
+         * @type {string}
+         * @default ''
+         */
+        this.origin = '';
+
+        /**
+         * The destination airpot of an aircraft
+         *
+         * This will only be populated for arrivals
```

Comment:
Not necessarily to be implemented now, i really just mean for reference to the comment.